packages.nix

```diff
@@ -52,7 +52,7 @@ in
   system-manager =
     pkgs.runCommand "system-manager"
       {
-        nativeBuildInputs = [ pkgs.makeBinaryWrapper ];
+        nativeBuildInputs = [ pkgs.pkgsBuildHost.makeBinaryWrapper ];
```

Comment:
Doesn't need to be addressed in this PR but generally I would use callPackage everywhere to make splicing work. cc @r-vdp @eureka-cpu